### PR TITLE
Clarify contribution guidelines for units

### PR DESCRIPTION
**Pull request by fabcor-maxiv**
Here is the preview: <https://mxcubecore--1064.org.readthedocs.build/en/1064/dev/contributing.html#units>.

### CODE REVIEW

**Comment by fabcor-maxiv** [comment]
Here is my attempt at clarifying the contribution guidelines for units...

**Comment by fabcor-maxiv** [comment]
Are we sure about "***translative*** motors" and "***rotative*** motors"? Isn't it rather translational and rotational?

**Comment by fabcor-maxiv** [comment]
Here is the preview: <https://mxcubecore--1064.org.readthedocs.build/en/1064/dev/contributing.html#units>.

**Review by elmjag — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -111,22 +111,33 @@ The exact routine is described more preceisly in [MEP01](https://github.com/mxcu
...
 When writing code that converts between different units,
-it is recommended to use utility functions from {py:mod}`mxcubecore.utils.units` module.
-This will to aid in the readability of the code.
+it is possible to use utility functions from {py:mod}`mxcubecore.utils.units` module,
+which will make the unit conversion explicit.
+
+Furthermore, it has been agreed that, whenever possible,
+the following units must be used across the code base:
+
+* mm (millimetre) for translative motors and sizes
```
Comment (rhfogh) [inline CONTRIBUTING.md]: I checked a dictionary - it is 'translational'. 'translative' means 'relating to transfer, or to translation'. Rotational should then be picked for symmetry (though here 'rotatory' might also do)

**Comment by rhfogh** [comment]
An additional change, while you are at it . In (or close to) line 195, teh sentende
"Contained HardwareObjects also use properties"
should probably be removed in the light of yaml configuration.

**Review by rhfogh — APPROVED** [review]
Very good.

**Comment by marcus-oscarsson** [comment]
Looks good :), you can merge when the discussed updates have been made. (Ill be away for the rest of the week :))

**Review by marcus-oscarsson — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (fabcor-maxiv) [reply CONTRIBUTING.md]: Done

**Comment by fabcor-maxiv** [comment]
> In (or close to) line 195, teh sentende
> "Contained HardwareObjects also use properties"
> should probably be removed in the light of yaml configuration.

I guess this should be done on the YAML branch then

**Pull request merged**